frontend/src/interfaces/parcel.ts

```diff
@@ -0,0 +1,21 @@
+import { IAddress, IBuilding, IEvaluation, IFiscal, IProperty } from 'interfaces';
+
+export interface IParcel extends IProperty {
+  pid?: string;
+  pin?: number | '';
```

Comment:
I'd love to find a way where we don't have to muddy our interfaces to support Formik/inputs `number | ''`